### PR TITLE
added performance_insights_enabled argument for rds_instance - #22

### PR DESCRIPTION
**Pull request by NishantTomar**
added the performance_insights_enabled argument to rds_instance and set the default value as false.
ISSUE- #22

### CODE REVIEW

**Comment by robg-test** [comment]
Hi @NishantTomar the workflow we have is running the command ``terraform fmt`` for linting.

Can you lint the changes on your side and push back up?

**Comment by NishantTomar** [comment]
@robg-test I ran the `terraform fmt`. And hope it passes the workflow this time.

**Comment by robg-test** [comment]
Hi @NishantTomar we have a small issue in regards to the workflow around forked repos. Will look at fixing it soon.

**Comment by robg-test** [comment]
Hi @NishantTomar we are a step closer to resolving this issue with the pipeline (@chrisbloe's merge)

**Comment by NishantTomar** [comment]
Hi @robg-test thanks for the update.
I see its still failing in workflow. Let me know if any changes are required.

**Comment by robg-test** [comment]
Hi again @NishantTomar , we've run into a bit of a security hitch around execution of this repo and forked repos.

The pipeline is meant to automatically commit documentation changes to the repo however this isn't working as your on a fork. Were going to have a look at next steps for supporting forks next week.

**Comment by NishantTomar** [comment]
Hi @robg-test is it fixed?

**Comment by robg-test** [comment]
Hi @NishantTomar 

So in order for forked pull requests to work we need to rethink some of the existing behavior around auto-commiting documentation changes. I will add this in scope for release 3 of these modules to come up with a fix for this.

I will replicate this in a PR branch directly on this repo.

Apologies, you were the first external editor of this repo and you've helped uncover a rather nasty bug in our workflow.

**Comment by robg-test** [comment]
See #93

File: modules/aws/rds_serverless_cluster/variables.tf [hunk 1]
```diff
@@ -101,3 +101,9 @@ variable "database_auto_minor_version_upgrade" {
 variable "prevent_deletion" {
   type = bool
 }
+
+variable "database_performance_insights_enabled" {
+  type        = bool
+  default     = false
+  description = "This is a boolean value determining if the rds instance have the performace insights enabled or not"
```
Comment (cmbuckley) [inline modules/aws/rds_serverless_cluster/variables.tf:108]:
```suggestion
  description = "Specifies whether the RDS instance has Performance Insights enabled."
```

**Review by cmbuckley — APPROVED** [review]

**Comment by cmbuckley** [comment]
Hi @NishantTomar, thank you for your PR! And sorry it took us so long to address the issue described above. I've merged a fix in #95 so documentation is updated automatically for all PRs.

**Comment by NishantTomar** [comment]
Hi @cmbuckley No problem. It was my pleasure to contribute to the repository.

**Pull request merged**